src/components/Table/GridTable.tsx

```diff
@@ -1239,12 +1256,13 @@ export function applyRowFn<R extends Kinded>(
   column: GridColumn<R>,
   row: GridDataRow<R>,
   api: GridTableApi<R>,
+  level?: number,
```

Comment:
Maybe make this required to make sure we don't have callers that we've forgotten to update?